website/src/middlewares/api-v3/index.js

```diff
@@ -1,10 +1,8 @@
-'use strict';
-
 // This module is only used to attach middlewares to the express app
 
-let errorHandler = require('./errorHandler');
+import errorHandler from './errorHandler';
 
-module.exports = function attachMiddlewares (app) {
+export default function middleware (app) {
```

Comment:
Ahh, didn't realize it had a name before. When I first worked on it, I believe it was an anonymous function. Will change.